src/pages/home/home.html

```diff
@@ -127,8 +125,60 @@
           </div>
         </ion-item-divider>
         <ion-list>
-          <page-wallet-activity *ngFor="let notification of notifications" (click)="openNotificationModal(notification)"
-            [notification]="notification"></page-wallet-activity>
+          <page-wallet-activity *ngFor="let notification of notifications" (click)="openNotificationModal(notification)" [notification]="notification"></page-wallet-activity>
+        </ion-list>
+      </div>
+
+      <div class="card" *ngIf="vaultWallets && vaultWallets[0]">
```

Comment:
We have repeated 3 times the code to list wallets in the same view:
We should optimize that in some way. For the moment, adding a 'TODO' would be enough, and we leave it for another PR

Reply:
done